projects/libreoffice/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -eu
+
+git pull -r
+
+#shuffle CXXFLAGS -stdlib=libc++ arg into CXX as well because we use
+#the CXX as the linker and need to pass -stdlib=libc++ to build
+export CXX="$CXX -stdlib=libc++"
+#similarly force the -fsanitize etc args in as well as pthread to get
+#things to link successfully during the build
+export LDFLAGS="$CFLAGS -lpthread"
+
+cd $OUT
+#build under $OUT cause its the only place without enough space to do so
+#build in a sub scratch dir so we can easily throw away the build artifacts
+#afterwards
+mkdir scratch
+cd scratch
+$SRC/libreoffice/autogen.sh --with-distro=LibreOfficeOssFuzz
+
+make build-nocheck
```

Comment:
--with-distro=LibreOfficeOssFuzz (newly) expands on our side to build additional fuzzing targets that assume they can link against -lFuzzingEngine

i.e. it builds this (initial) target https://cgit.freedesktop.org/libreoffice/core/tree/vcl/workben/wmffuzzer.cxx?id=65cc20c6911bdbe0995691d6c3d0f5c772155d13 with this makefile https://cgit.freedesktop.org/libreoffice/core/tree/vcl/Executable_wmffuzzer.mk?id=65cc20c6911bdbe0995691d6c3d0f5c772155d13